include/real/integer_number.hpp

```diff
@@ -283,33 +279,33 @@ namespace boost{
              */
 
 
-			integer<T> &operator=(const integer<T>& other) = default;
-			integer<T> operator-(void) {
-				integer result = (*this);
+			integer_number<T> &operator = (const integer_number<T>& other) = default;
```

Comment:
I particularly prefer having the symbol next to the operator keyword, like operator-, operator+, but I guess is just preference, if we using it like this everywhere we should keep it this way.

Reply:
I have used it like this everywhere.